.github/workflows/graph-develop.yaml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Graph to develop
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
+        run: yarn --frozen-lockfile
+      - name: Lint
+        run: yarn lint
+
+  deploy-staging-gnosis:
+    runs-on: ubuntu-latest
+    env:
+      GRAPH_SUBGRAPH_ACCOUNT: ${{ github.repository_owner }}
+      GRAPH_SUBGRAPH_NAME: giveth-economy-goerli-staging
+      GRAPH_CONFIG_FILE: subgraph.deployment-7-goerli.yaml
+      NETWORK_NAME: goerli
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install
+        run: yarn --frozen-lockfile
+      - uses: gtaschuk/graph-deploy@v0.1.12
+        with:
+          graph_access_token: ${{ secrets.GRAPH_ACCESS_TOKEN }}
+          graph_subgraph_name: $GRAPH_SUBGRAPH_NAME
+          graph_account: $GRAPH_SUBGRAPH_ACCOUNT
+          graph_config_file: $GRAPH_CONFIG_FILE
```

Comment:
I don't see the reason to use env here, only `GRAPH_ACCESS_TOKEN` is secret. I suggest removing all env vars and writing the values directly here.

Reply:
For reference, this is how I would like it https://github.com/balancer-labs/balancer-subgraph-v2/blob/master/.github/workflows/graph.yml#L24